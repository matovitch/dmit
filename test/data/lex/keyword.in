if
else
let
func
while
export
import
module
return