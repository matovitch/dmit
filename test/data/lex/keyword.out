{"tokens" :["START_OF_INPUT","IF","WHITESPACE","ELSE","WHITESPACE","LET","WHITESPACE","FUNC","WHITESPACE","WHILE","WHITESPACE","EXPORT","WHITESPACE","IMPORT","WHITESPACE","MODULE","WHITESPACE","RETURN","END_OF_INPUT"],"offsets":[50,48,47,43,42,39,38,34,33,28,27,21,20,14,13,7,6,0,0]}